<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlightModeConfig</class>
 <widget class="QWidget" name="FlightModeConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;h2&gt;Flight Modes&lt;/h2&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>71</y>
     <width>456</width>
     <height>178</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Flight Mode 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Flight Mode 2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Flight Mode 3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Flight Mode 4</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Flight Mode 5</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Flight Mode 6</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QComboBox" name="mode1ComboBox">
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="mode2ComboBox">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="mode3ComboBox"/>
      </item>
      <item>
       <widget class="QComboBox" name="mode4ComboBox"/>
      </item>
      <item>
       <widget class="QComboBox" name="mode5ComboBox"/>
      </item>
      <item>
       <widget class="QComboBox" name="mode6ComboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="mode1SimpleCheckBox">
        <property name="text">
         <string>Simple Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mode2SimpleCheckBox">
        <property name="text">
         <string>Simple Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mode3SimpleCheckBox">
        <property name="text">
         <string>Simple Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mode4SimpleCheckBox">
        <property name="text">
         <string>Simple Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mode5SimpleCheckBox">
        <property name="text">
         <string>Simple Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mode6SimpleCheckBox">
        <property name="text">
         <string>Simple Mode</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="mode1Label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>PWM 0 - 1230</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mode2Label">
        <property name="text">
         <string>PWM 1231 - 1360</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mode3Label">
        <property name="text">
         <string>PWM 1361 - 1490</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mode4Label">
        <property name="text">
         <string>PWM 1491 - 1620</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mode5Label">
        <property name="text">
         <string>PWM 1621 - 1749</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mode6Label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>PWM 1750 +</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>267</y>
     <width>95</width>
     <height>14</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>14</height>
    </size>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="format">
    <string>%p%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savePushButton">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>259</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
